pkg/influxunifi/influxdb.go

```diff
@@ -22,22 +23,35 @@ import (
 const PluginName = "influxdb"
 
 const (
-	defaultInterval   = 30 * time.Second
-	minimumInterval   = 10 * time.Second
-	defaultInfluxDB   = "unifi"
-	defaultInfluxUser = "unifipoller"
-	defaultInfluxURL  = "http://127.0.0.1:8086"
+	defaultInterval     = 30 * time.Second
+	minimumInterval     = 10 * time.Second
+	defaultInfluxDB     = "unifi"
+	defaultInfluxUser   = "unifipoller"
+	defaultInfluxOrg    = "unifi"
+	defaultInfluxBucket = "unifi"
+	defaultInfluxURL    = "http://127.0.0.1:8086"
 )
 
 // Config defines the data needed to store metrics in InfluxDB.
 type Config struct {
-	Interval  cnfg.Duration `json:"interval,omitempty" toml:"interval,omitempty" xml:"interval" yaml:"interval"`
-	URL       string        `json:"url,omitempty" toml:"url,omitempty" xml:"url" yaml:"url"`
-	User      string        `json:"user,omitempty" toml:"user,omitempty" xml:"user" yaml:"user"`
-	Pass      string        `json:"pass,omitempty" toml:"pass,omitempty" xml:"pass" yaml:"pass"`
-	DB        string        `json:"db,omitempty" toml:"db,omitempty" xml:"db" yaml:"db"`
-	Disable   bool          `json:"disable" toml:"disable" xml:"disable,attr" yaml:"disable"`
-	VerifySSL bool          `json:"verify_ssl" toml:"verify_ssl" xml:"verify_ssl" yaml:"verify_ssl"`
+	Interval cnfg.Duration `json:"interval,omitempty" toml:"interval,omitempty" xml:"interval" yaml:"interval"`
+	Version2 bool          `json:"version2,omitempty" toml:"version2,omitempty" xml:"version2" yaml:"version2"`
```

Comment:
Can you poll influx to figure this out?

Reply:
Perhaps you can determine if V2 based on the inputs?
If username and password then v1. If auth token, then v2.

Reply:
this might be the simplest way yes